src/application-listing/ApplicationList.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { Form } from 'reactstrap';
+import Table from 'rc-table';
+import './application-listing.css';
+
+class ApplicationList extends React.Component {
+  constructor (props) {
+    super(props);
+    const { applications } = props;
```

Comment:
If this component ever receives new applications as props for some reason, the component won't update as we're duplicating applications in state. Is there any reason why we can't use applications in props the whole time?